docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md

```diff
@@ -1,107 +1,46 @@
 # MATCH
 
-`MATCH`语句提供基于模式（pattern）匹配的搜索功能。
+`MATCH`语句提供基于模式（Pattern）匹配的搜索功能，其通过定义一个或多个模式，允许在 NebulaGraph 中查找与模式匹配的数据。在检索到匹配的数据后，用户可以使用 RETURN 子句将其作为结果返回。
 
-一个`MATCH`语句定义了一个搜索模式，用该模式匹配存储在 NebulaGraph 中的数据，然后用`RETURN`子句检索数据。
-
-本文示例使用测试数据集 [basketballplayer](../1.nGQL-overview/1.overview.md#basketballplayer) 进行演示。
+在本文中，我们将使用名为 [basketballplayer](../1.nGQL-overview/1.overview.md#basketballplayer) 的测试数据集来演示 MATCH 语句的使用。
 
 ## 语法
 
-与`GO`或`LOOKUP`等其他查询语句相比，`MATCH`的语法更灵活。`MATCH`语句采用的路径类型是[`trail`](../../1.introduction/2.1.path.md)，即遍历时只有点可以重复，边不可以重复。
+`MATCH`语句的语法相较于其他查询语句（如`GO`和`LOOKUP`）更具灵活性。在进行查询时，`MATCH`语句使用的路径类型是[`trail`](../../1.introduction/2.1.path.md)，这意味着点可以重复出现，但边不能重复。
 
-`MATCH` 语法如下：
+`MATCH`语法的基本结构如下：
 
 ```ngql
 MATCH <pattern> [<clause_1>]  RETURN <output>  [<clause_2>];
 ```
 
-- `pattern`：pattern 的详细说明请参见[模式](../1.nGQL-overview/3.graph-patterns.md)。`MATCH`语句支持匹配一个或多个模式，多个模式之间用英文逗号（,）分隔。例如`(a)-[]->(b),(c)-[]->(d)`。
+- `pattern`：`MATCH`语句支持匹配一个或多个模式，多个模式之间用英文逗号（,）分隔。例如`(a)-[]->(b),(c)-[]->(d)`。Pattern 的详细说明请参见[模式](../1.nGQL-overview/3.graph-patterns.md)。
 
 - `clause_1`：支持`WHERE`、`WITH`、`UNWIND`、`OPTIONAL MATCH`子句，也可以使用`MATCH`作为子句。
 
-- `output`：定义需要返回的输出。可以使用`AS`设置输出的别名。
+- `output`：定义需要返回输出结果的列表名称。可以使用`AS`设置列表的别名。
 
 - `clause_2`：支持`ORDER BY`、`LIMIT`子句。
 
-## 使用限制
-
-未创建[索引](../14.native-index-statements/1.create-native-index.md)时，仅以下情况支持使用`MATCH`查询。当使用`MATCH`语句出现报错时，用户可以创建并重建索引后执行查询语句。
-
-- `MATCH`语句中`WHERE`子句使用 id() 函数指定了点的 VID，不需要创建索引即可执行。
-
-- 当遍历所有点或边时，例如`MATCH (v) RETURN v LIMIT N`、`MATCH ()-[e]->() RETURN e LIMIT N`。
-
-- 当遍历指定 Tag 的点，例如`MATCH (v:player) RETURN v LIMIT N`。
-
-- 当遍历指定 Edge Type 并指定边的方向，例如`MATCH ()-[e:follow]->() RETURN e LIMIT N`。
-
-!!! note
-
-    - 目前 MATCH 语句无法查询到悬挂边。
-    - 不支持没有索引时同时遍历指定的 Tag 和 Edge Type，例如执行`MATCH (v:player)-[e:follow]->() RETURN e LIMIT N`会报错。
 
 !!! compatibility "历史版本兼容性" 
 
-    从 3.0.0 版本开始，为了区别不同 Tag 的属性，返回属性时必须额外指定 Tag 名称。即从`RETURN <变量名>.<属性名>`改为`RETURN <变量名>.<Tag名>.<属性名>`。
+    - 从 3.5.0 版本开始，`MATCH`语句支持全表扫描，即在不使用任何索引或者过滤条件的情况下可遍历图中点或边。在此之前的版本中，MATCH 语句在某些情况下需要索引才能执行查询或者需要使用`LIMIT`限制输出结果数量。
+    - 从 3.0.0 版本开始，为了区别不同 Tag 的属性，返回属性时必须额外指定 Tag 名称。即从`RETURN <变量名>.<属性名>`改为`RETURN <变量名>.<Tag名>.<属性名>`。
 
 
+## 使用说明
 
-## 示例
```

Comment:
Why delete these?

Reply:
> Why delete these?

 just changed this name to `use patterns`